Makefile

```diff
@@ -1,9 +1,13 @@
 init: Pipfile
+	pip install pipenv --user
+	pipenv install --python 3
+
+dev-init: Pipfile
 	pip install --user pipenv
-	pipenv install
+	pipenv install --python 3 --dev
```

Comment:
We support only python 3 now - why do we need to specify --python 3?
Maybe you install pipenv in python2.7? If you install the pipenv package
"pipenv install" just works.